pkgs/tools/misc/git-town/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }: 
+
+let
+  version = "7.2.0";
+in
+  buildGoPackage rec {
+    name = "git-town-${version}";
+    version = "7.2.0";
+
+    goPackagePath = "github.com/Originate/git-town";
+
+    src = fetchFromGitHub {
+      owner = "Originate";
+      repo = "git-town";
+      rev = "v${version}";
+      sha256 = "0hr0c6iya34lanfhsg9kj03l4ajalcfxkbn4bgwh0749smhi6mrj";
+    };
+
+    buildFlagsArray = ''
+    -ldflags=-X github.com/Originate/git-town/src/cmd.version=v${version} -X github.com/Originate/git-town/src/cmd.buildDate=nix
+    '';
+
+    meta = {
+      description = "Generic, high-level git support for git-flow workflows";
+      homepage = http://www.git-town.com/;
+      maintainers = [ "Originate" "allonsy" ];
```

Comment:
`maintainers` should be a list of attributes of `stdenv.lib.maintainers`.
Add yourself to https://github.com/nixos/nixpkgs/blob/master/maintainers/maintainer-list.nix in a separate commit if you want to be a maintainer for this package.

Reply:
Yeah, I'll add myself as a maintainer.
We've got a few options:
I can add myself to maintainers.txt in this PR although it seems like you want a separate PR.
If that's true I can submit a separate PR with that change and we can either
1) wait for that PR to get committed
2) I can remove the maintainers clause in the default.nix and then add it in later when I get added as a maintainer

Thoughts?